src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerMetricsDiagnosticListener.cs

```diff
@@ -31,21 +32,24 @@ internal sealed class HttpHandlerMetricsDiagnosticListener : ListenerHandler
 {
     internal const string OnStopEvent = "System.Net.Http.HttpRequestOut.Stop";
 
+    internal static readonly AssemblyName AssemblyName = typeof(HttpClientMetrics).Assembly.GetName();
+    internal static readonly string MeterName = AssemblyName.Name;
+    internal static readonly string MeterVersion = AssemblyName.Version.ToString();
+    internal static readonly Meter Meter = new(MeterName, MeterVersion);
+    private static readonly Histogram<double> HttpClientDuration = Meter.CreateHistogram<double>("http.client.duration", "ms", "Measures the duration of outbound HTTP requests.");
```

Comment:
@TimothyMothra I am asuming this change will not work for you? As we need to create histograms based on the `HttpSemanticConventions` flag.